src/System.Net.HttpListener/src/System/Net/Managed/HttpListenerRequest.Managed.cs

```diff
@@ -643,6 +642,34 @@ public bool IsWebSocketRequest
         {
             get
             {
```

Comment:
Just for context, did you write this code, or is this copied from somewhere? What about other parts of this PR?